OneCommunityGlobal/HighestGoodNetworkApp
# Yixiao_TaskEditSuggestionRow_unittest

# Description
Create a unit test for the TaskEditSuggestionRow component to confirm whether it working as intended.


## Main changes explained:
- Create file TaskEditSuggestionRow.test.js to do the unit test.
- Change some structure of file TaskEditSuggestionRow.jsx for more friendly test.

## How to test:
1. check into current branch
2. do `npm install`  to run this PR locally
3. run `npm test TaskEditSuggestionRow.test.js`


## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49748716/26b27985-334f-4f2a-9946-286cae324cea)
